test/JDBC/expected/SYSTEM_USER-vu-verify.out

```diff
@@ -0,0 +1,31 @@
+-- tsql
+SELECT session_user, system_user, current_user, db_name();
+GO
+~~START~~
+nvarchar#!#nvarchar#!#varchar#!#nvarchar
+dbo#!#jdbc_user#!#dbo#!#master
+~~END~~
+
+
+-- tsql user=system_user_vu_prepare_r1 password=123
+SELECT session_user, system_user, current_user, db_name();
+GO
+~~START~~
+nvarchar#!#nvarchar#!#varchar#!#nvarchar
+guest#!#system_user_vu_prepare_r1#!#guest#!#master
+~~END~~
+
+
+-- tsql user=system_user_vu_prepare_r2 password=123
+SELECT session_user, system_user, current_user, db_name();
+GO
+~~START~~
+nvarchar#!#nvarchar#!#varchar#!#nvarchar
+guest#!#system_user_vu_prepare_r2#!#guest#!#master
+~~END~~
```

Comment:
I think we need to create some function/procedure/view that depends on the object you want to test.

Reply:
Created a procedure that depends on System_user and session_user object which needs to be tested.

Reply:
we should have view/function/procedure all three. View is a must as function and procedures do not create dependency, they get recompiled so you can drop and recreate objects inside them